chrome/application-services-navigation.json

```diff
@@ -67,10 +67,28 @@
                 }
             ]
         },
+        {
+            "groupId": "insights",
```

Comment:
You are missing the icon property. Based on the mocks, it should be
```js
"icon": "trend-up"
```

Reply:
Added.